### PR TITLE
concrete-strength-test

### PR DESCRIPTION
**Pull request by kirikaran**

### CODE REVIEW

File: tokyo-supermix-server/src/main/java/com/tokyo/supermix/server/controller/ConcreteStrengthTestController.java [hunk 1]
```diff
@@ -1,10 +1,109 @@
...
+
+
+  @GetMapping(value = EndpointURI.CONCRETE_STRENGTH_TEST_BY_ID)
+  public ResponseEntity<Object> getConcreteStrengthTestById(@PathVariable Long id) {
+    if (concreteStrengthTestService.isConcreteStrengthTestExist(id)) {
+      logger.debug("Get ConcreteStrengthTest By Id");
+      ConcreteStrengthTest concreteStrengthTest =
+          concreteStrengthTestService.getConcreteStrengthTestById(id);
+      return new ResponseEntity<>(new ContentResponse<>(Constants.CONCRETE_STRENGTH_TEST,
+          mapper.map(concreteStrengthTest, ConcreteStrengthTestResponseDto.class),
```
Comment (Keerthana65) [inline tokyo-supermix-server/src/main/java/com/tokyo/supermix/server/controller/ConcreteStrengthTestController.java]: avoid the creating object, here you can directly pass.
 for example mapper.map(concreteStrengthTestService.getConcreteStrengthTestById(id), ConcreteStrengthTestResponseDto.class),

File: tokyo-supermix-server/src/main/java/com/tokyo/supermix/server/controller/ConcreteStrengthTestController.java [hunk 2]
```diff
@@ -1,10 +1,109 @@
...
+          HttpStatus.OK);
+    }
+    return new ResponseEntity<>(
+        new ValidationFailureResponse(Constants.CONCRETE_STRENGTH_TEST_ID,
+            validationFailureStatusCodes.getConcreteStrengthTestNotExist()),
+        HttpStatus.BAD_REQUEST);
+  }
+
+  @PutMapping(value = EndpointURI.CONCRETE_STRENGTH_TEST)
+  public ResponseEntity<Object> updateCustomer(
```
Comment (Mathuragini) [inline tokyo-supermix-server/src/main/java/com/tokyo/supermix/server/controller/ConcreteStrengthTestController.java]: Update method name change to -updateConcreteStrengthTest

File: tokyo-supermix-server/src/main/java/com/tokyo/supermix/server/controller/ConcreteStrengthTestController.java [hunk 3]
```diff
@@ -1,10 +1,109 @@
...
+          RestApiResponseStatus.OK), HttpStatus.OK);
+    }
+    return new ResponseEntity<>(
+        new ValidationFailureResponse(Constants.CONCRETE_STRENGTH_TEST_ID,
+            validationFailureStatusCodes.getConcreteStrengthTestNotExist()),
+        HttpStatus.BAD_REQUEST);
+  }
+
+  @DeleteMapping(value = EndpointURI.CONCRETE_STRENGTH_TEST_BY_ID)
+  public ResponseEntity<Object> deleteCustomer(@PathVariable Long id) {
```
Comment (Mathuragini) [inline tokyo-supermix-server/src/main/java/com/tokyo/supermix/server/controller/ConcreteStrengthTestController.java]: change method name to deleteConcreteStrengthTest

File: tokyo-supermix-server/src/main/resources/ValidationMessages.properties [hunk 4]
```diff
@@ -166,3 +166,14 @@ projectRequestDto.contactNumber.empty=58007
 projectRequestDto.plantCode.null=58008
 projectRequestDto.plantCode.empty=58009
  
+#validation for ConcreteStrengthTest
+validation.concreteStrengthTest.notExists=70000
+concreteStrengthTestRequestDto.mixDesignCode.null=70001
+concreteStrengthTestRequestDto.mixDesignCode.empty=70002
+ConcreteStrengthTestRequestDto.plantCode.null=70003
```
Comment (Mathuragini) [inline tokyo-supermix-server/src/main/resources/ValidationMessages.properties]: use camel case

**Pull request merged**